kv/raftstore/peer_storage.go

```diff
@@ -508,10 +439,15 @@ func (ps *PeerStorage) SaveReadyState(ready *raft.Ready) (*ApplySnapResult, erro
 	kvWB, raftWB := new(engine_util.WriteBatch), new(engine_util.WriteBatch)
 	ctx := NewInvokeContext(ps)
```

Comment:
actually, we can remove the invoke context. Let's leave it to future work

Reply:
I had removed it, PTAL.